integration-tests/test/rpc.spec.ts

```diff
@@ -207,6 +207,36 @@ describe('Basic RPC tests', () => {
         'Contract creation code contains unsafe opcodes. Did you use the right compiler or pass an unsafe constructor argument?'
       )
     })
+
+    it('should allow calls with nonzero value, even from accounts with no balance', async () => {
+      // Deploy a contract to check msg.value of the call
+      const Factory__ValueContext: ContractFactory = await ethers.getContractFactory(
+        'ValueContext',
+        wallet
+      )
+      const ValueContext: Contract = await Factory__ValueContext.deploy()
+      await ValueContext.deployTransaction.wait()
+
+      // make sure we do not have funds at this account already
+      const from = ethers.constants.AddressZero
+      const fromBalance = await wallet.provider.getBalance(from)
+      expect(fromBalance).to.deep.eq(
+        BigNumber.from(0),
+        'from account does not have 0 for its balance, did another test fund it?'
```

Comment:
Plz add a comment here explaining that this is a meta-failure with the integration tests and not with this specific functionality.